src/logo/builtin.c

```diff
@@ -1340,6 +1340,16 @@ static const FFlogo E[] = {
         .colorKeys = FF_COLOR_FG_BLUE,
         .colorTitle = FF_COLOR_FG_WHITE,
     },
+    // evolinx
+    {
+        .names = {"Evolinx", "evolinx", },
```

Comment:
fastfetch matches os name case-insensitively. `"evolinx"` is not necessary here.

Reply:
* Accidnetally saved test edits from kwrite so made changes

```
{"Evolinx", "evolinx", }, -> {"Evolinx"},
```